Orange/canvas/report/report.py

```diff
@@ -216,7 +223,7 @@ def data(role=Qt.DisplayRole,
                     if row is None or col is None:
                         return model.headerData(col if row is None else row,
                                                 orientation, role)
-                    return model.data(model.index(row, col), role)
+                    return fmtnum(model.data(model.index(row, col), role))
```

Comment:
Would you care to write this as:
```py
val = model.data(model.index(row, col), role)
return try_(lambda: num_format.format(float(val)), val)
```
?
`try_`, an ingenious invention of mine, does exactly what [this try-except](https://github.com/kernc/orange3/pull/4/files#diff-a8517f05dab824c1e209188aaf6661d5R202) does.